File: src/main/java/com/example/demo/service/iml/LivoRecomentService.java
package com.example.demo.service.iml;

import com.example.demo.dao.ILivoRecomentDao;
import com.example.demo.entity.LivoRecoment;
import com.example.demo.service.ILivoRecomentService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.List;
@Service
public class LivoRecomentService implements ILivoRecomentService {
    @Autowired
    ILivoRecomentDao liloRecommitDado;
    @Override
    public List<LivoRecoment> GetRecomeData(LivoRecoment mlivoRecom) {
        return liloRecommitDado.GetLoveData();
    }
}

File: src/main/java/com/example/demo/service/iml/LivoRecomentService.kt
package com.example.demo.service.iml

import com.example.demo.dao.ILiveRecoupmentDao
import com.example.demo.entity.LiveRecoupment
import com.example.demo.service.ILiveRecoupmentService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class LiveRecoupmentService : ILiveRecoupmentService {
    @Autowired
    lateinit var liloRecommitDado: ILiveRecoupmentDao
    override fun recoupmentList(liveRecoupment: LiveRecoupment): List<LiveRecoupment> {
        return liloRecommitDado.getLiveListData()
    }
}
